airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -727,3 +730,12 @@ def get_cli_commands() -> list[GroupCommand]:
                 subcommands=KUBERNETES_COMMANDS,
             )
         ]
+
+
+def get_parser() -> argparse.ArgumentParser:
```

Comment:
Should this be something that goes into the BaseExecutor? It looks generic and we'll need it for all executors that have cli commands.

Reply:
Thought about it. Maybe. The small problem with it ist that this is a code that might be installed on earlier version of airflow. BUT it's only executed during sphinx docs build, so yeah, it is pretty safe to extract common code. On it.

Reply:
Done.